application/service_sync/service_sync.py

```diff
@@ -69,19 +71,19 @@ def remove_sqs_message_from_queue(event: UpdateRequestQueueItem) -> None:
     logger.info("Removed SQS message from queue", extra={"receipt_handle": event["recipient_id"]})
 
 
-# @metric_scope
-# def add_success_metric(event: UpdateRequestQueueItem, metrics) -> None:  # type: ignore
-#     """Adds a success metric to the custom metrics collection
+@metric_scope
+def add_success_metric(event: UpdateRequestQueueItem, metrics) -> None:  # type: ignore
+    """Adds a success metric to the custom metrics collection
 
-#     Args:
-#         event (UpdateRequestQueueItem): Lambda function invocation event
-#     """
-#     after = time_ns() // 1000000
-#     metadata: UpdateRequestMetadata = event["metadata"]
-#     message_received = metadata["message_received"]
-#     diff = after - message_received
-#     metrics.set_namespace("UEC-DOS-INT")
-#     metrics.set_property("level", "INFO")
-#     metrics.set_property("message", f"Recording change request latency of {diff}")
-#     metrics.put_metric("QueueToDoSLatency", diff, "Milliseconds")
-#     metrics.set_dimensions({"ENV": environ["ENV"]})
+    Args:
+        event (UpdateRequestQueueItem): Lambda function invocation event
+    """
+    after = time_ns() // 1000000
+    metadata: UpdateRequestMetadata = event["metadata"]
+    message_received = metadata["message_received"]
+    diff = after - message_received
+    metrics.set_namespace("UEC-DOS-INT")
+    metrics.set_property("level", "INFO")
+    metrics.set_property("message", f"Recording change request latency of {diff}")
+    metrics.put_metric("QueueToDoSLatency", diff, "Milliseconds")
```

Comment:
Not sure what `message_received` is in but `after` is in nano seconds. Wonder if the diff metric is in nanoseconds  rather than milliseconds?